apps/aechannel/src/aesc_fsm.erl

```diff
@@ -1345,7 +1425,7 @@ error_binary(E) when is_atom(E) ->
     atom_to_binary(E, latin1).
 
 
-terminate(Reason, _State, Data) ->
+terminate(Reason, _State, #data{session = _Session} = Data) ->
```

Comment:
Why this change? `_Session` doesn't seem to be referred to.

Reply:
Refactoring side effect